tests/test_unit/test_cdf_tk/test_commands/test_dump_data.py

```diff
@@ -95,6 +99,69 @@ def test_dump_assets(self, tmp_path: Path) -> None:
         parquet_files = list(parquet_dir.rglob("*.parquet"))
         assert len(parquet_files) == 1
 
+    def test_dump_filemetadata(self, tmp_path: Path) -> None:
+        my_label = LabelDefinition(external_id="label1", name="Label 1")
+        dataset = DataSet(external_id="my_dataset", name="My Dataset", id=123)
+        my_file = FileMetadata(
+            external_id="my_file",
+            name="My File",
+            data_set_id=dataset.id,
+            metadata={"key": "value"},
+            source="MySource",
+            labels=[Label(external_id=my_label.external_id)],
+            asset_ids=[1],
+            mime_type="application/octet-stream",
+            directory="my_directory",
+            geo_location=GeoLocation(
+                type="Feature",
+                geometry=Geometry(type="LineString", coordinates=[[1.0, 1.0], [2.0, 2.0], [3.0, 3.0]]),
+                properties={},
+            ),
+        )
+        cmd = DumpDataCommand(skip_tracking=False, print_warning=False)
+        output_dir = tmp_path / "file_dump"
+        with monkeypatch_toolkit_client() as client:
+            client.files.return_value = [FileMetadataList([my_file])]
+            client.files.aggregate.return_value = [CountAggregate(count=1)]
+            client.labels.retrieve.return_value = LabelDefinitionList([my_label])
+            client.data_sets.retrieve_multiple.return_value = [dataset]
+            client.lookup.assets.external_id.return_value = "rootAsset"
+            client.lookup.data_sets.external_id.return_value = dataset.external_id
+            client.transformations.preview.return_value = TransformationPreviewResult(
+                None, [{"key": "key", "key_count": 1}]
+            )
+
+            cmd.dump_table(
+                FileMetadataFinder(client, ["rootAsset"], []),
+                output_dir,
+                clean=True,
+                limit=None,
+                format_="csv",
+                verbose=False,
+            )
+
+        output_csvs = list(output_dir.rglob("*.csv"))
+        assert len(output_csvs) == 1
+        output_csv = output_csvs[0]
```

Comment:
Can you have a test case here for output_parquet as well? Looks like it is missing.

Reply:
That would be testing the TableWrites and not the functionality in this PR. That is tested extensively in https://github.com/cognitedata/toolkit/blob/main/tests/test_unit/test_cdf_tk/test_utils/test_table_writers.py